### PR TITLE
Unify the name of using

### PR DESCRIPTION
**Pull request by xiaocai2333**
fix issue: https://github.com/milvus-io/milvus/issues/6721
Signed-off-by: xiaocai2333 <cai.zhang@zilliz.com>

### CODE REVIEW

File: pymilvus_orm/collection.py [hunk 1]
```diff
@@ -107,6 +110,7 @@ def __init__(self, name, schema=None, **kwargs):
             0
         """
         self._name = name
+        self.using = using
```
Comment (scsven) [inline pymilvus_orm/collection.py:113]: Can this variable be renamed to `_using`?

Comment (xiaocai2333) [reply pymilvus_orm/collection.py:113]: ok

**Pull request merged**